File: WPA.backend/WPA.backend/Entities/Expense.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace WPA.backend.Entities
{
    public class Expense
    {
        public int Id { get; set; }
        public int PlannerId { get; set; }
        public int? ServiceProviderId { get; set; }
        public ServiceProvider? ServiceProvider { get; set; }
        public string Name { get; set; }
        public decimal Amount { get; set; }
        public string Adnotation { get; set; }
        public decimal Deposit { get; set; }
        public ExpenseStatus ExpenseStatus { get; set; }
    }
}

File: frontend/src/app/api/models/expense.ts
/* tslint:disable */
import { ExpenseStatus } from './expense-status';
import { ServiceProvider } from './service-provider';
export interface Expense {
  adnotation?: string;
  amount: number;
  deposit: number;
  expenseStatus: ExpenseStatus;
  id: number;
  name?: string;
  plannerId: number;
  provider?: string;
  serviceProvider?: ServiceProvider;
  serviceProviderId?: number;
}
